src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exeption.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+import mate.jdbc.util.RemoteConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private final ConnectionUtil connectionDB = new RemoteConnectionUtil();
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertRequest = "INSERT INTO manufacturers(name, country) VALUES(?, ?);";
+        try (Connection connection = connectionDB.getConnect();
+                 PreparedStatement createManufacturer =
+                         connection.prepareStatement(insertRequest,
+                             Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturer.setString(1, manufacturer.getName());
+            createManufacturer.setString(2, manufacturer.getCountry());
+            createManufacturer.executeUpdate();
+            ResultSet resultSet = createManufacturer.getGeneratedKeys();
+            if (resultSet.next()) {
+                manufacturer.setId(resultSet.getObject("id", Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert manufacturer to DB", e);
```

Comment:
```suggestion
            throw new DataProcessingException("Can't insert manufacturer to DB " + manufacturer , e);
```
Используй информативные сообщения. Подправь еще где нужно по коду.